RevenueCat/Plugins/Android/com/revenuecat/purchasesunity/purchasesWrapper.kt

```diff
@@ -0,0 +1,12 @@
+package com.revenuecat.purchasesunity;
+
+import com.revenuecat.purchases.hybridcommon.mappers.convertToJson
+import com.revenuecat.purchases.hybridcommon.setLogHandler
+
+private const val HANDLE_LOG = "_handleLog"
+
+fun setLogHandler() {
+    setLogHandler {
+        PurchasesWrapper.sendJSONObject(it.convertToJson(), HANDLE_LOG)
+    }
+}
```

Comment:
This is an alternative to use the new function added in https://github.com/RevenueCat/purchases-hybrid-common/pull/328

Adding Kotlin works fine, the only problem I have seen is that after exporting the project, it adds Kotlin 1.4.31, and trying to compile the project gives an error that requires updating to 1.6.0. I think it's due to incompatibilities either with the Gradle version or the Android Gradle Plugin.

Do you think it's a deal breaker?

Reply:
cc @RevenueCat/coresdk

Reply:
Hmm would this affect developers in their own projects? If so, I think it would be.

This incompatibility is between the unity sdk and the `purchases-android` SDK? If so, I guess that means our android SDK is not compatible with that old version of kotlin...

Reply:
I think it's between purchases-android or purchases-hybrid-common, I am not sure, but yes, I think it's in general an incompatibility between the version of the kotlin build plugin and the version we used to compile

Reply:
And yes, this would mean people exporting their unity project with this version of Unity, which enables Kotlin 1.4.31 need to update their build.gradle

Reply:
We could also try to make our android sdk compatible with that version of kotlin... but not sure if that's possible

Reply:
By the way, I tested Android's MagicWeather and looks like we require Kotlin 1.5.0+, using a kotlin version lower than that gives the same error